LoginAndAclSetup.md

```diff
@@ -0,0 +1,178 @@
+  * Select **Add a permission**, and then **My APIs**.
+  * In the list of applications, select your server application **Azure Search OpenAI Demo API**
+  * Ensure **Delegated permissions** is selected.
+  * In the **Select permissions** section, select the **access_as_user** permission
+  * Select **Add permissions**.
+
+#### Configure Server App Known Client Applications
+
+Consent from the user must be obtained for use of the client and server app. The client app can prompt the user for consent through a dialog when they log in. The server app has no ability to show a dialog for consent. Client apps can be [added to the list of known clients](https://learn.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow#gaining-consent-for-the-middle-tier-application) to access the server app, so a consent dialog is shown for the server app.
+
+* Navigate to the server app registration
+* In the left hand menu, select **Manifest**
+* Replace `"knownClientApplications": []` with `"knownClientApplications": ["<client application id>"]`
+* Select **Save**
+
+#### Testing
+
+Ensure you run `azd env set AZURE_USE_AUTHENTICATION` to enable the login UI once you have setup the two Azure AD apps before you deploy or run the application. The login UI will not appear unless all [required environment variables](#environment-variables-reference) have been setup.
+
+In both the chat and ask a question modes, an optional **Use oid security filter** and **Use groups security filter** will appear. The oid (User ID) filter maps to the `oids` field and the groups (Group ID) filter maps to the `groups` field in the search index. Use the optional scripts included in the sample to manage values for these fields.
+
+It's possible that your tenant admin has placed a restriction on consent to apps with [unverified publishers](https://learn.microsoft.com/azure/active-directory/develop/publisher-verification-overview). In this case, only admins may consent to the client and server apps, and normal user accounts are unable to use the login system until the admin consents on behalf of the entire organization.
+
+## Optional scripts
+
+Two optional scripts are provided that allow easier setup of sample data with document level access control.
+
+### Azure Data Lake Storage Gen2 Setup
+
+[Azure Data Lake Storage Gen2](https://learn.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-introduction) implements an [access control model](https://learn.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-access-control) that can be used for document level access control. The [adlsgen2setup.ps1](./scripts/adlsgen2setup.ps1) script uploads the sample data included in the [data](./data) folder to a Data Lake Storage Gen2 storage account.
+
+In order to use this script, an existing Data Lake Storage Gen2 storage account. Run `azd env set AZURE_ADLS_GEN2_STORAGE_ACCOUNT <your-storage-account>` prior to running the script.
```

Comment:
Do you need to set up roles/permissions to be able to access the account locally?

Reply:
Correct - adding this to the doc